### PR TITLE
Adds wdio-image-comparison-service Visual Regression Testing service to website

### PR DESCRIPTION
**Pull request by anselmbradford**
## Proposed changes

Adds wdio-image-comparison-service Visual Regression Testing service to website

## Types of changes

- [ ] Documentation update

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by anselmbradford** [comment]
I have to sign a license agreement to update the docs?

**Comment by christian-bromann** [comment]
@anselmbradford any code contribution requires to sign a CLA. This is common practice across many OSS projects.

File: scripts/docs-generation/3rd-party/services.json [hunk 1]
```diff
@@ -52,5 +52,11 @@
     "title": "LambdaTest",
     "githubUrl": "https://github.com/LambdaTest/wdio-lambdatest-service",
     "npmUrl": "https://www.npmjs.com/package/wdio-lambdatest-service"
+  },
+  {
+    "packageName": "wdio-image-comparison-service",
+    "title": "Visual Regression Testing",
+    "githubUrl": "https://github.com/wswebcreation/wdio-image-comparison-service",
```
Comment (emilyrohrbough) [inline scripts/docs-generation/3rd-party/services.json:59]: Should this be added if the the tool used for comparison is still being developed? https://github.com/wswebcreation/webdriver-image-comparison

Comment (wswebcreation) [reply scripts/docs-generation/3rd-party/services.json:59]: @emilbryggare 

The webdriver-image-comparison is still being developed, it's the core of the comparison module, but can't be used as a standalone module together with WebdriverIO

**Pull request closed without merging**